src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -431,7 +438,36 @@ def parse_actions_from_repositories(allow_or_remove_repository):
 
     return actions
 
+def prepare_source_location(source_location, client_registries, registry_name, resource_group_name):
+    if source_location.lower() == NULL_CONTEXT:
```

Comment:
null check for source_location

Reply:
She has a null check around the task run call. As you stated earlier, we don't need a null check for all uses of this method. In that case, it's best to let the caller handle the null check.

Reply:
I'm good with that. That comment is missing a question mark. I was just wondering if it is better to have it on the function now that is gonna be in the utils folder.